chromaui/addon-visual-tests
# Keep the "create first build" UI right up until the running build has captured this story

We plan to add a in progress indicator to this: https://linear.app/chromaui/issue/AP-3621#comment-1b9a109b, but in the meantime these stories just make sure the selected builds mirror what we want to happen.

@ghengeveld can you let me know if you want a separate ticket for the "first build" in progress UI linked above, or if you just plan to tackle it as part of what you are doing now.

## To test:

1. Create a new branch
2. You should see the "create first build" screen
3. Start the build
4. The screen should remain there (with button disabled) until there is a captured test for the current story.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.74--canary.89.65ca55a.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.74--canary.89.65ca55a.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.74--canary.89.65ca55a.0
  ```
</details>
